pytorch_lightning/utilities/xla_device_utils.py

```diff
@@ -38,10 +39,13 @@ def pl_multi_process(func):
     @functools.wraps(func)
     def wrapper(*args, **kwargs):
         queue = Queue()
-        proc = Process(target=inner_f, args=(queue, func,), kwargs=kwargs)
+        proc = Process(target=inner_f, args=(queue, func, *args), kwargs=kwargs)
         proc.start()
-        proc.join()
-        return queue.get()
+        proc.join(10)
+        try:
+            return queue.get_nowait()
+        except q.Empty:
+            return False
```

Comment:
@awaelchli should we raise an exception here instructing the user to set up TPU's properly? i.e adding `TPU_IP_ADDRESS`

Reply:
maybe, could there be other  reasons for a timeout? is there also a TPU_MASTER_PORT or similar?

Reply:
Nah. Don't see any ENV var with that name.